src/plugins/yanlr/listener.cpp

```diff
@@ -132,6 +132,7 @@ void KeyListener::enterPair (PairContext * context)
 	{
 		// Add key with empty value
 		// The parser does not visit `exitValue` in that case
+		child.setBinary (NULL, 0);
```

Comment:
Maybe we even add a convenience function setNull in the C++ binding?

Reply:
I do not know, if it makes that much sense to add a new method, since we might not use it that often. On the other hand, I would certainly have used the method `setNull`, if it would exist.